source/switch_storage_engines.rst

```diff
@@ -13,81 +13,87 @@ option when you start ``mongod``.  Alternatively, you can set the
+     $ rm -rf <dbpathDataDir>
+     $ # Update the configuration file by setting the new
+     $ # value for the storage.engine variable
+     $ # set the engine-specific settings such as
+     $ # storage.inMemory.engineConfig.inMemorySizeGB
      $ service mongod start
+     
+  Wait for the node to rejoin with the other relicset members and report the
+  SECONDARY status. 
 
-- If there is data that you want to migrate and make compatible with the new
-  storage engine, use the ``mongodump`` and ``mongorestore`` utilities:
+  Repeat the procedure on the remaining nodes.
+    
+* for a standalone instance or a single-node replicaset, use the
+  ``mongodump`` and ``mongorestore`` utilities:
 
   .. code-block:: bash
 
      $ mongodump --out <dumpDir>
-     $ service mongod stop
-     $ rm -rf /var/lib/mongodb/*
-     $ sed -i '/engine: .*inMemory/s/#//g' /etc/mongod.conf
+     $ rm -rf <dbpathDataDir>
+     $ # Update the configuration file by setting the new
+     $ # value for the storage.engine variable
+     $ # set the engine-specific settings such as
+     $ # storage.wiredTiger.engineConfig.cacheSizeGB or
+     $ # storage.inMemory.engineConfig.inMemorySizeGB
      $ service mongod start
      $ mongorestore <dumpDir>
-
-Storage Engine Configuration Templates
+ 
+Data at Rest Encryption
 ================================================================================
 
-The default configuration file provided with |PSMDB| (:file:`/etc/mongod.conf`)
-contains templates for running any of the bundled storage engine.  The suggested
-values do not cover all cases, but they are a good reference point to start
-customizing the configuration.  The following hints may help you:
-
-- On very high volume systems, you may need to increase the maximum number of
-  concurrent transactions for WiredTiger_ and :ref:`inmemory` (default is 128).
-  These are controlled using the following MongoDB parameters:
-
-  - ``wiredTigerConcurrentReadTransactions``
-  - ``wiredTigerConcurrentWriteTransactions``
-
-- On systems with more than roughly 24,000 collections running MMAPv1_,
-  increase the ``storage.mmapv1.nsSize`` variable.
-
-- Also for MMAPv1_ you can set ``storage.mmapv1.smallFiles`` to ``true`` to use
-  a smaller default file size, if you have a large number of databases that each
-  holds a small amount of data.
-
-
-- The default maximum internal cache size that WiredTiger_ uses
-  is the larger of either:
-
-  - 50% of RAM minus 1 GB
-  - 256 MB
+Using :ref:`psmdb.data-at-rest-encryption` means using the same ``storage.*``
+configuration options as for WiredTiger_. To change from normal to :ref:`psmdb.data-at-rest-encryption`
+mode or backward, you must clean up the ``dbPath`` data directory,
+just as if you change the storage engine. This is becase
+|mongod| cannot convert the data files to an encrypted format 'in place'. It
+mostly gets the document data again either via the initial sync from another
```

Comment:
```suggestion
must get the document data again either via the initial sync from another
```
I would bet this was my mistake in that google doc. Sorry about that - Akira

Reply:
Fixed